source/includes/markdown/_18-webhooks.html.md

```diff
@@ -235,6 +235,16 @@ that webhook will no longer be available.
   <img src="../images/webhook-heartbeat-failed.png" alt="Failed webhook heartbeat (After initial handshake)"/>
 </a>
 
+# Webhook Security
+
+In order to receive webhook events, your webhook endpoint needs to be accessible over the internet. Because your webhook endpoint is publicly accessible over the internet, it is vulnerable to receiving events that are not from Asana. To ensure that your webhook endpoint receives events from Asana, it is important to verify each request.
```

Comment:
Hmm how about changing 

> Because your webhook endpoint is publicly accessible over the internet...

to 

> As such, because your webhook endpoint is publicly accessible...